platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheKey.kt

```diff
@@ -91,6 +92,16 @@ class ConfigurationCacheKey(
         )
     }
 
+    private
+    fun Hasher.putDevelocityUrl() {
+        val develocityUrl = startParameter.develocityUrl
+        if (develocityUrl != null) {
+            putString(develocityUrl)
+        } else {
+            putNull()
```

Comment:
🚀 `putNull` is unnecessary, the absence of `putString` is enough to change the hash